src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -331,3 +331,16 @@ def load_arguments(self, _):
     with self.argument_context('monitor activity-log alert scope remove') as c:
         c.argument('scopes', options_list=['--scope', '-s'], nargs='+')
     # endregion
+
+    # region Log Analytics Workspace
+    with self.argument_context('monitor log-analytics workspace') as c:
+        from azure.mgmt.loganalytics.models import SkuNameEnum
+        c.argument('location', get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('workspace_name', options_list=['--workspace-name', '-n'], help="Name of the Log Analytics Workspace.")
+        c.argument('sku', deprecate_info=c.deprecate(hide=True), arg_type=get_enum_type([SkuNameEnum.per_gb2018.value]), help="The SKU of the workspace.", default=SkuNameEnum.per_gb2018.value)
```

Comment:
Please remove deprecate_info and define the default value in your method. In addition, you need to ignore sku argument.